marshmallow/schema.py

```diff
@@ -797,12 +800,17 @@ def _invoke_field_validators(self, data, many):
                         field_obj=field_obj
                     )
 
-    def _invoke_validators(self, pass_many, data, original_data, many):
+    def _invoke_validators(self, pass_many, data, original_data, many, field_errors):
```

Comment:
In order to maintain backwards-compat, could you make `field_errors` an optional argument?